YANG/tapi-common.yang

```diff
@@ -144,10 +144,32 @@ module tapi-common {
                 uses lifecycle-state-pac-g;
                 description "none";
             }
+            container capacity {

```

Comment:
I think it would be better to add the TransferCapacityPac to SIP/ConnSEP rather than add Capacity, because TransferCapacityPac has total capacity and available capacity which is what I proposed.